docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "2"
+services:
+
+  backend:
+    build: .
+    command: tail -f /dev/null
+    environment:
+      DJANGO_SETTINGS_MODULE: pah-fm_org.settings.dev
```

Comment:
Do we use these variables?

Reply:
As You haven't initialize Django project none of these variables is usable here.